### DISCUSSION TITLE
eslint **/*.ts{,x} - error vs eslint **/*.ts **/*.tsx - works fine

### DISCUSSION THREAD

**Discussion by KERIST** [eslint/eslint#15764] [Q&A] [answered]
Hello! I'm using ubuntu 20.04.4 LTS
yarn version - 1.22.18
npm version - 8.5.5
node version - v17.8.0
The problem is when I run `yarn lint` with this setup
`"lint": "tsc --noEmit && eslint **/*.ts{,x}",`
I got a lot of errors, but if change extensions everything work smoothly
`"lint": "tsc --noEmit && eslint **/*.ts **/*.tsx",`
Does everyone know what might cause this issue?

**Comment by nzakas** [comment, marked as answer]
The shell expansion is being done by the operating system for these commands. I’m not 100% sure, but I think the first case is an invalid pattern.

**Reply by KERIST** [reply]
Thank you so much for your help!
